### DISCUSSION TITLE
How to create bundle on Mac

### DISCUSSION THREAD

**Discussion by markdemich** [airsdk/Adobe-Runtime-Support#2496] [Q&A]
I have successfully created a bundle for windows, but my mac bundle is giving me an error "output file is not writable".  Here's a snippet of my acconfig.json

```
	"signingOptions": {
		"storetype": "pkcs12",
		"keystore": "cert.pfx"
	},
        "windows": {
            "target": "bundle",
            "output": "bin/Deploy/windows"
        },
       "mac": {
	    "target": "bundle",
	    "output": "bin/Deploy/mac"
	}
```
I also tried bin/Deploy/my-application.app instead of bin/Deploy/mac and I get the same error.  

Any ideas.

**Comment by markdemich** [comment]
I was able to get this working.  It was the relative path.  If I simply put "output" : "xyz.app", it worked, but it didn't put the .app file where I want it.  Any way to fix that?

**Reply by ajwfrost** [reply]
It should work if the folder is already there..? so if you did `mkdir -p bin/Deploy` and then ran ADT with that configuration, you'd successfully generate `bin/Deploy/mac.app`

We can probably fix this though, just by doing a "mkdirs" on the parent of the output file...

**Comment by markdemich** [comment]
Thanks.  First, I'm embarrassed I hadn't figured that out on my own.  It
was a long day though and I'm a windows guy.

However, fixing this would be good since my bin folder in not in source
control so anyone pulling the repo down will need to know to create this
folder before packaging.  I guess I can add a "dependsOn" task in VSCode to
make sure the directory exists, so there are workarounds for now.

**Reply by ajwfrost** [reply]
> fixing this would be good 

Agreed, and a fairly straightforward fix to make here.. will be in the next release.
thanks
